Makefile.in

```diff
@@ -40,6 +48,27 @@ $(PROG): $(OBJECTS)
 .c.o: config.h
 	$(CC) $(CPPFLAGS) $(CFLAGS) -c $< -o $@
 
+%.gz: %
+ifeq ($(GZIP),)
+	$(error No gzip compression available)
+else
+	$(GZIP) -c $< > $@
+endif
+
+%.bz2: %
+ifeq ($(BZIP2),)
+	$(error No bzip2 compression available)
+else
+	$(BZIP2) -c $< > $@
+endif
+
+%.xz: %
+ifeq ($(XZ),)
+	$(error No xz compression available)
+else
+	$(XZ) -c $< > $@
+endif
```

Comment:
Well, with that this Makefile is not POSIX anymore. We have explicitly removed GNU extensions with Olaf back then to make it POSIX only.

Reply:
The latest changeset converts those to shell conditionals, and adds an explicit `.POSIX:` statement. So syntax seems be fine, and current build rules should be unaffected. Only the `dist` targets will require a `make` implementation with suitable extensions, but then building the source distribution is architecture independent by definition.